hilti/include/ast/declarations/module.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
+
+#pragma once
+
+#include <utility>
+
+#include <hilti/ast/declaration.h>
+#include <hilti/ast/expression.h>
+#include <hilti/ast/id.h>
+#include <hilti/ast/module.h>
+#include <hilti/base/result.h>
```

Comment:
Let's include all required headers.
```suggestion

#include <hilti/ast/declaration.h>
#include <hilti/ast/expression.h>
#include <hilti/ast/id.h>
#include <hilti/ast/meta.h>
#include <hilti/ast/module.h>
#include <hilti/ast/node.h>
#include <hilti/ast/node_ref.h>
#include <hilti/base/result.h>
#include <hilti/base/type_erase.h>
```